twopaneview/examples/package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "ReactNativeDualScreenExample",
+  "version": "0.0.1",
+  "private": true,
+  "scripts": {
+    "start": "react-native start",
+    "lint": "../node_modules/.bin/tslint -p ./src",
+    "reset:packager": "watchman watch-del-all && react-native start --reset-cache",
+    "run:android": "react-native run-android --no-jetifier",
+    "clean:android": "cd android && ./gradlew clean",
+    "install:pods": "cd ios && which pod && pod install || echo '\\033[1;33m Warning: cocoapods not found!\\033[0m'",
+    "generate:android:signing-key": "keytool -genkey -v -keystore example.keystore -alias example -keyalg RSA -keysize 2048 -validity 10000 && mv example.keystore android/app",
+    "generate:android:apk": "npm run generate:android:bundle && cd android && ./gradlew assembleRelease",
+    "generate:android:bundle": "npm run build && react-native bundle --platform android --dev false --entry-file index.js --bundle-output ./android/app/src/main/assets/index.android.bundle"
+  },
+  "dependencies": {
+    "@react-native-community/masked-view": "^0.1.6",
+    "react": "16.9.0",
+    "react-native": "0.61.5",
+    "react-native-dualscreen": "file:../../dualscreeninfo",
```

Comment:
>file:../../dualscreeninfo [](start = 32, length = 25)

This should be an npm version, not a file dependency, and the package name is react-native-dualscreeninfo.  You can use verdaccio to test publishing/consuming react-native-dualscreeninfo.  See [https://github.com/verdaccio/verdaccio](https://github.com/verdaccio/verdaccio) for more info.

Reply:
the npm package for react-native-dualscreen is not up to date.
Interesting, never used verdaccio before, will give it a try.

Reply:
I filed [https://github.com/microsoft/react-native-dualscreen/issues/6](https://github.com/microsoft/react-native-dualscreen/issues/6) to track publishing react-native-dualscreeninfo.  Hopefully this can be parallelized.  Let me know if you get blocked by this.

---
In reply to: [392522452](https://github.com/microsoft/react-native-dualscreen/pull/5#discussion_r392522452) [](ancestors = 392522452)